addons/common/functions/fnc_execRemoteFnc.sqf

```diff
@@ -20,6 +20,8 @@
  */
 #include "script_component.hpp"
 
+ACE_DEPRECATED("ace_common_fnc_execRemoteFnc","3.6.0","ace_common_fnc_globalEvent");
```

Comment:
I'd say 3.6 is far enough, looking at previous releases it's quite some time between minor (eg. 3.4 -> 3.5) version updates. I think it's safe.